client-v2/src/util/delete.ts

```diff
@@ -0,0 +1,17 @@
+export const deleteIssue = () => {
+  const issue = window.location.href.substring(
+    window.location.href.lastIndexOf('/') + 1
+  );
```

Comment:
`window.location.href` may not work on all browsers, you can add to README like use chrome or so